### PR TITLE
feature(198): add stubString

### PR DESCRIPTION
**Pull request by BjornMelgaard**

### CODE REVIEW

File: src/index.d.ts [hunk 1]
```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
          */
         stubNull():  null;
 
+        /**
+         * A function that returns new empty string.
```
Comment (char0n) [inline src/index.d.ts]: A function that returns empty string.

File: src/index.d.ts [hunk 2]
```diff
@@ -284,6 +284,11 @@ declare namespace RamdaAdjunct {
...
+         */
+        stubString(): '';
```
Comment (char0n) [inline src/index.d.ts]: Again, infering type not the value ?

File: src/stubString.js [hunk 3]
```diff
@@ -0,0 +1,19 @@
+import { always } from 'ramda';
+
+/**
+ * A function that returns new empty string.
```
Comment (char0n) [inline src/stubString.js]: A function that returns empty string.

File: src/stubString.js [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.1.0|v2.1.0}
+ * @category Function
+ * @sig ... -> ''
+ * @return {''}
+ * @example
+ *
+ * RA.stubString(); //=> ''
+ * RA.stubString(1, 2, 3); //=> ''
+ */
+const stubString = always('');
```
Comment (char0n) [inline src/stubString.js]: This is OK because in JS two empty strings are always equal

File: test/stubString.js [hunk 5]
```diff
@@ -0,0 +1,11 @@
+import * as RA from '../src/index';
+import eq from './shared/eq';
+
+describe('stubString', function() {
+  it("tests `function` that returns `'''`", function() {
+    eq(RA.stubString(), '');
+    eq(RA.stubString([1]), '');
+    eq(RA.stubString(new Array()), '');
+    eq(RA.stubString(1, 2, 3), '');
+  });
```
Comment (char0n) [inline test/stubString.js]: context('when called', function() {
  specify('should always return empty string', function() {
 ....test here for equality of results of 2 consequent invocations
  });
});

File: src/index.d.ts (hunk 2 above)
Comment (BjornMelgaard) [reply src/index.d.ts]: Is this a problem, typescript handles this fine?

Comment (char0n) [reply src/index.d.ts]: If it handles it, it's fine. Did not know about literal types in typescript.

File: src/stubString.js [hunk 6]
```diff
@@ -0,0 +1,19 @@
+import { always } from 'ramda';
+
+/**
+ * A function that returns empty string.
+ *
+ * @func stubString
+ * @memberOf RA
+ * @since {@link https://char0n.github.io/ramda-adjunct/2.1.0|v2.1.0}
+ * @category Function
+ * @sig ... -> ''
```
Comment (char0n) [inline src/stubString.js]: Hindley-Milner signatures don't use literal types. They explusively use types. It's deal with that

Comment (BjornMelgaard) [reply src/stubString.js]: @char0n you merged it with wrong signature `@sig ... -> ''`?

Comment (char0n) [reply src/stubString.js]: yeah, fixed it here: https://github.com/char0n/ramda-adjunct/commit/555ae2b30c29ae3cdb98b4dfe6b2c33ab4d4617f

**Pull request merged**